app/src/main/java/org/jellyfin/androidtv/ui/playback/overlay/CustomPlaybackTransportControlGlue.java

```diff
@@ -242,6 +250,9 @@ void addMediaActions() {
                 secondaryActionsAdapter.add(recordAction);
                 recordingStateChanged();
             }
+        } else {
+            secondaryActionsAdapter.add(selectQualityAction);
+            secondaryActionsAdapter.add(selectAudioOutputAction);
```

Comment:
I think these would look nicer if positioned after the playbackSpeedAction.
Or at least the skipNextAction should be the first button on the lower row.

Reply:
Resolved via commit: https://github.com/jellyfin/jellyfin-androidtv/pull/1406/commits/546e77034d3bc901b1225188bfd9739d05a665fd